utils/command_build_clib.py

```diff
@@ -64,7 +64,7 @@
 _project_linker = "dpcpp"
 _project_cmplr_flag_sycl_devel = ["-fsycl-device-code-split=per_kernel"]
 _project_cmplr_flag_sycl = ["-fsycl"]
-_project_cmplr_flag_stdcpp_static = ["-static-libstdc++", "-static-libgcc"]
+_project_cmplr_flag_stdcpp_static = [] # This brakes TBB ["-static-libstdc++", "-static-libgcc"]
```

Comment:
The comment is slightly misleading. It does not brake TBB but it brakes C++ runtime behavior because exception are improperly handled due to symbols conflicts. I.e. the exception is created and handled in one instance of C++ runtime but `std::current_exception` (called by TBB in a catch section) is called from another instance C++ runtime. Therefore, the `std::current_exception` returns empty exception (because the second C++ runtime does not know about exception created by the first C++ runtime) that crashes C++ runtime if passed to `std::rethrow_exception`.